pkg/pillar/cmd/waitforaddr/waitforaddr.go

```diff
@@ -31,38 +31,51 @@ const (
 var Version = "No version specified"
 
 // Context for handleDNSModify
-type DNSContext struct {
+type waitForAddrContext struct {
 	deviceNetworkStatus    types.DeviceNetworkStatus
 	usableAddressCount     int
 	DNSinitialized         bool // Received DeviceNetworkStatus
 	subDeviceNetworkStatus pubsub.Subscription
+	agentBaseContext       *agentbase.Context
+	// cli options
+	versionPtr *bool
+	noPidPtr   *bool
 }
 
-var debug = false
-var debugOverride bool // From command line arg
-var logger *logrus.Logger
+func newWaitForAddrContextContext(_ *pubsub.PubSub, logger *logrus.Logger, log *base.LogObject, arguments []string) *waitForAddrContext {
```

Comment:
"Context" is twice in the function name.

Reply:
Modified, thanks